README.md

```diff
@@ -1 +1,87 @@
-# Technical-Task-Challenges
\ No newline at end of file
+# Technical Task Challenges
+
+Virtual Coffee's Technical Task Challenges are created/aggregated by VC members to help other VC members gain practical experience in how to use technical skills in real-world scenarios.
+
+### Benefits
+
+These task challenges will greatly benefit the entire Virtual Coffee community. They will serve our members who are current working professionals by allowing them to up-skill in select technologies and tools. They will also help our job seekers, especially those with no prior professional experience in the tech industry. The unique added value of the technical task challenges is that they're modeled after real-world scenarios, lending these exercises a quality that far surpasses what's readily available on sites like LeetCode and Codewars.
```

Comment:
**thought:** I think we should move this to become the first section of the **Table of Contents**, retitle it, and reword it a bit. When we original wrote it, we were trying to convince people why this would be a good idea. Now, people will already be at the repo when they see it so I think we should change it to present tense and clean it up a little. We can do this now or open a new issue to do it in a separate PR.